tests/di/test_di_durability.py

```diff
@@ -21,44 +21,47 @@
 F-23B : Data Durability/Integrity test module.
 """
 
-import os
 import logging
-import pytest
+import os
 import secrets
 from time import perf_counter_ns
+
+import pytest
 from boto3.s3.transfer import TransferConfig
+
+from commons.constants import MB, KB
 from commons.constants import NORMAL_UPLOAD_SIZES
-from commons.constants import NORMAL_UPLOAD_SIZES_IN_MB
-from commons.utils import assert_utils
-from commons.utils import system_utils
-from commons.exceptions import CTException
+from commons.constants import const
 from commons.ct_fail_on import CTFailOn
 from commons.errorcodes import error_handler
+from commons.exceptions import CTException
 from commons.helpers.health_helper import Health
 from commons.params import TEST_DATA_FOLDER, VAR_LOG_SYS
-from commons.constants import const
-from commons.constants import MB, KB
+from commons.utils import assert_utils
+from commons.utils import system_utils
 from config import CMN_CFG
 from config.s3 import S3_BLKBOX_CFG
-from libs.s3 import S3_CFG
+from libs.di import di_lib
+from libs.di.data_generator import DataGenerator
 from libs.di.di_error_detection_test_lib import DIErrorDetection
 from libs.di.di_feature_control import DIFeatureControl
-from libs.di.data_generator import DataGenerator
 from libs.di.fi_adapter import S3FailureInjection
-from libs.di import di_lib
-from libs.s3.s3_test_lib import S3TestLib
-from libs.s3.s3_multipart_test_lib import S3MultipartTestLib
-from libs.s3 import cortxcli_test_lib
-from libs.s3.s3_cmd_test_lib import S3CmdTestLib
-from libs.s3.s3_blackbox_test_lib import JCloudClient
+from libs.s3 import S3_CFG
 from libs.s3 import SECRET_KEY, ACCESS_KEY
+from libs.s3 import cortxcli_test_lib
 from libs.s3 import s3_s3cmd
+from libs.s3.s3_blackbox_test_lib import JCloudClient
+from libs.s3.s3_cmd_test_lib import S3CmdTestLib
+from libs.s3.s3_multipart_test_lib import S3MultipartTestLib
+from libs.s3.s3_test_lib import S3TestLib
 
 
+# pylint: disable=too-many-instance-attributes, too-many-public-methods
```

Comment:
can we have pylint disables on 2 seperate lines?

Reply:
done